### DISCUSSION TITLE
中英混搭怎么对齐

### DISCUSSION THREAD

**Discussion by hhfwork** [BYJRK/DotNet-Discussions#194] [C# 技术交流]
有没有同时中英等距的字体，毕竟工作中有些时候会中英混搭，配合Cleanup会错位，就很烦
![微信截图_20231228143710](https://github.com/BYJRK/DotNet-Discussions/assets/138421386/041b8b6b-5c7f-43f8-9910-c5a3aef81ab2)

原问题：有没有同时中英等距的字体，比如中文宽度是英语的两倍？

**Comment by BYJRK** [comment]
[更纱黑体](https://github.com/be5invis/Sarasa-Gothic)

完全符合你的需求，西文字母宽度是 CJK 的一半

**Reply by hhfwork** [reply]
好吧确实是满足了，但是还是错位，而且丑拒 :joy:
![微信截图_20231229075447](https://github.com/BYJRK/DotNet-Discussions/assets/138421386/c949b4a7-63e8-4f23-af6a-70ac11b75f04)

我在找找是不是Cleanup的问题吧.

**Comment by lilongyan666** [comment]
Maple Mono SC NF
![PixPin_2024-03-01_18-41-20](https://github.com/BYJRK/DotNet-Discussions/assets/56418643/15d6f233-e610-46a6-b86b-adc4f680f758)
